composer.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "yiisoft/yii",
+	"name": "kf99916/yii",
```

Comment:
I'm sorry about that. Please ignore changes in composer.json. My project is dependent on Yii 1.1.17, however, there is a issue about sorting of CArrayDataProvider. Reviewing pull request and releasing next version needs time. In order to ensure my project can move on the next version smoothly, so I let my project be dependent on my repo.